indigo_social/badges.py

```diff
@@ -140,7 +140,7 @@ class ContributorBadge(PermissionBadge):
     slug = 'contributor'
     name = 'Contributor'
     description = 'Can view work details'
-    permissions = ('indigo_api.view_work',)
+    permissions = ('indigo_api.view_work', 'indigo_workflow.view_bulklistimportworksprocess')
```

Comment:
Eish, this is starting to blur the lines between indigo_social and indigo_workflow.

Reply:
would it be better to change the permissions for seeing the tasks?